tools/developer/run.sh

```diff
@@ -44,34 +43,34 @@ TAP_DIRS="dijkstra"
 
 function set_cmake {
     # Using all defaults
-    #cmake ..
+    #cmake -DPOSTGRESQL_PG_CONFIG="/usr/lib/postgresql/${PGVERSION}/bin/pg_config" ..
```

Comment:
These are examples, The usage of this file is:
cp the file to your repository root
Make all the adjustments you need to do depending on your system
 
if in your particular system you need to add the 
`-DPOSTGRESQL_PG_CONFIG="/usr/lib/postgresql/${PGVERSION}/bin/pg_config"`
do it in the copy of the file in the root of the repository

Keep the examples as they are.
Just add a new example that "informs the location of pg_config" together with a comment